kubernetes/charts/direktiv/templates/prometheus-rolebind-cluster.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
```

Comment:
do we need this permission at all?

Reply:
Metrics don't appear to be gathered without this ClusterRoleBinding.